tests/test_dataset.py

```diff
@@ -776,3 +776,13 @@ def test_Dataset_from_url(urlopen):
     assert ds.bibpath == 'http://example.org/ds1.bib'
     assert ds.bibname == 'ds1.bib'
     assert len(ds.sources) == 3
+
+
+def test_Dataset_get_foreign_key_target(tmp_path):
+    ds = StructureDataset.in_dir(tmp_path)
+    ds.add_component('LanguageTable')
+    t, c = ds.get_foreign_key_target('values.csv', 'Language_ID')
+    assert ds.get_tabletype(t) == 'LanguageTable'
+    assert c.name == 'ID'
+
+    assert ds.get_foreign_key_target('values.csv', 'Value') is None
```

Comment:
@Anaphory here's how you would use the new method. Does this fit your bill?

Reply:
I tend to work more with components/properties, instead of urls/names. I can easily get them, though, so it's not a big hurdle.

Reply:
Ah, you are using pycldf index operations anyway.

Reply:
Ah yes. This isn't documenting best practice (which is what you follow :) ) - but testing the worst case scenario of people using file names, etc. :)
It highlights, though, that this also works for non-CLDF foreign keys.

Reply:
Anyway, `ds.get_foreign_key_target('ValueTable', 'languageReference')` will work as well.